src/main/webapp/assets/cwnedit.js

```diff
@@ -133,14 +134,18 @@ function addsense(context, pos="",synonym="",definition="",abbrev="",misspell=""
                             </tr>
                         <table>
                         <div id="relation{{id}}-help" class="cwn-help">
-                            <p>Please give at least one relation linking this term to an existing term in Colloquial or Princeton WordNet by typing the word under target and selecting the synset from the autocomplete. In general, it is expected that nouns always have a hypernym (broader) term, verbs, adjectives and adverbs may have a broader term and/or a similar term. Please also consider the origin of the term: if it is derived from an existing word, please add a "derived from" link, if this word is borrowed from another language, please add a loanword linking, whose target is the synset for the language this word is borrowed from</p>
+                            <p>Please give at least one relation linking this term to an existing term in Colloquial or Princeton WordNet by typing the word under target and selecting the synset from the autocomplete. In general, it is expected that nouns always have a hypernym (broader) term, verbs, adjectives and adverbs may have a broader term and/or a similar term. Please also consider the origin of the term: if it is derived from an existing word, please add a "derived from" link, if this word is borrowed from another language, please add a loanword linking, whose target is the synset for the language this word is borrowed from.</p>
+                            <p>Common relations:</p>
```

Comment:
I've split this into two lists following the division in the current dropdowns.
We ought to explain the connection and update if the dropdowns change.